<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1161</width>
    <height>1012</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>1</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #2979FF</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="lbl_imagenUat">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>0</y>
      <width>331</width>
      <height>121</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../../Archivos/Recursos.qrc">:/Logos/Archivos/log_uat_nuevo.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_imagenFi">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>0</y>
      <width>211</width>
      <height>141</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../../Archivos/Recursos.qrc">:/Logos/Archivos/FIT_logo_vertical.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_titulo">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>140</y>
      <width>1051</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Forte</family>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
      <underline>true</underline>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">text-decoration: underline;
font: 20pt &quot;Forte&quot;;
color:rgb(0, 0, 0);
border-radius: 10px;
font-weight: bold;
font-style: italic;</string>
    </property>
    <property name="text">
     <string>Dadas tres opciones (nombres de personajes, nombres de comida, etc),</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_Unidad2">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>100</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">text-decoration: underline;
font: 20pt &quot;Forte&quot;;
color:rgb(0, 0, 0);
border-radius: 10px;
font-weight: bold;
font-style: italic;</string>
    </property>
    <property name="text">
     <string>Unidad 2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_Jugar">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>220</y>
      <width>181</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Forte</family>
      <pointsize>30</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 30pt &quot;Forte&quot;;
background-color: rgb(130, 0, 2);
color: rgb(255, 255, 255);
border-radius:30px;
border:6px solid #B0BEC5;</string>
    </property>
    <property name="text">
     <string>Jugar</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_titulo_2">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>180</y>
      <width>981</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Forte</family>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
      <underline>true</underline>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">text-decoration: underline;
font: 20pt &quot;Forte&quot;;
color:rgb(0, 0, 0);
border-radius: 10px;
font-weight: bold;
font-style: italic;</string>
    </property>
    <property name="text">
     <string>escoger aquella opción a la que se refiera la imagen presentada</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_titulo_3">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>210</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Forte</family>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
      <underline>true</underline>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">text-decoration: underline;
font: 20pt &quot;Forte&quot;;
color:rgb(0, 0, 0);
border-radius: 10px;
font-weight: bold;
font-style: italic;</string>
    </property>
    <property name="text">
     <string>en cada ocasión</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_imagenGatos">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>780</y>
      <width>311</width>
      <height>171</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../../Archivos/Recursos.qrc">:/Logos/Archivos/export202502092301563482.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_1">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>670</y>
      <width>301</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Forte</family>
      <pointsize>20</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 20pt &quot;Forte&quot;;
background-color: rgb(130, 0, 2);
color: rgb(255, 255, 255);
border-radius:30px;
border:6px solid #B0BEC5;</string>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_imagen_2">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>320</y>
      <width>311</width>
      <height>321</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>40</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Imagen</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_2">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>670</y>
      <width>301</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Forte</family>
      <pointsize>20</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 20pt &quot;Forte&quot;;
background-color: rgb(130, 0, 2);
color: rgb(255, 255, 255);
border-radius:30px;
border:6px solid #B0BEC5;</string>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_3">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>670</y>
      <width>301</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Forte</family>
      <pointsize>20</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 20pt &quot;Forte&quot;;
background-color: rgb(130, 0, 2);
color: rgb(255, 255, 255);
border-radius:30px;
border:6px solid #B0BEC5;</string>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1161</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="../../Archivos/Recursos.qrc"/>
  <include location="../../Archivos/Recursos.qrc"/>
 </resources>
 <connections/>
</ui>
